class UserQuit(Exception): pass
class MainLoopError(Exception): pass